src/pages/library/Datasets.js

```diff
@@ -164,24 +164,24 @@ const amppd = () => h(Participant, {
     p(['Includes data from the following studies:']),
     div({ style: { margin: '0.4rem 0', fontWeight: 'bold', lineHeight: '150%' } }, [
       div({ style: { display: 'flex' } }, [
-        div({ style: { width: 410 } }, [`• Parkinson's Disease Biomarkers Program (PDBP)`]),
+        div({ style: { width: 410 } }, ['• Parkinson\'s Disease Biomarkers Program (PDBP)']),
         div(['• BioFIND'])
       ]),
       div({ style: { display: 'flex' } }, [
-        div({ style: { width: 410 } }, [`• Parkinson's Progression Markers Initiative (PPMI)`]),
+        div({ style: { width: 410 } }, ['• Parkinson\'s Progression Markers Initiative (PPMI)']),
         div(['• Harvard Biomarkers Study (HBS)'])
       ]),
       div({ style: { display: 'flex' } }, [
-        div({ style: { width: 410 } }, [`• Lewy Body Dementia Study (LBD)`]),
+        div({ style: { width: 410 } }, ['• Lewy Body Dementia Study (LBD)']),
         div(['• LRRK2 Cohort Consortium (LCC)'])
       ]),
       div({ style: { display: 'flex' } }, [
-        div({ style: { width: 410 } }, [`• STEADY-PD3 Study`]),
+        div({ style: { width: 410 } }, ['• STEADY-PD3 Study']),
         div(['• SURE-PD3 Study'])
       ])
     ]),
     p([
-      `Global Parkinson's Genetics Program (GP2) data is accessible through an approved single AMP PD-GP2 DUA, `,
+      'Global Parkinson\'s Genetics Program (GP2) data is accessible through an approved single AMP PD-GP2 DUA, ',
```

Comment:
Wow, the need to escape an apostrophe comes up more than I would have expected. Personally, this is why I would like the flexibility to use double-quotes for strings.

Reply:
@kyuksel can you toss a quick poll in the terra-ui channel to make sure the majority of folks want this change? Sorry to be picky, but given the wide usage of back ticks in the code I think it's best to make sure everyone is on the same page.

Reply:
There is an `avoidEscape` option that would allow double quotes when the string contains single quotes.

https://eslint.org/docs/latest/rules/quotes#avoidescape

Reply:
@kyuksel Maybe we just do a quick check-in during today's mob time? One could say it's somewhat related… I imagine we will talk about how to maintain consistency in the codebase with less oversight (and how important we feel consistency is).